superset/fission/api.py

```diff
@@ -68,6 +69,12 @@ def get(self, path, **kwargs: Any) -> Response:
                                                                           os.environ.get('FISSION_SCOPE'),
                                                                           'superset',
                                                                           cache_result=True)
+        alfred_instance = os.environ.get('ALFRED_ENV')
+        if alfred_instance:
+           request_url = request.url + f'&alfred_instance={alfred_instance}'
+           logger.info(request_url)
+        else:
+           logger.info('ALFRED_ENV environment variable not set')
```

Comment:
could you remove these lines. I think they were removed from cccs-2.1 after you branched off so I think that is why they are still in your branch